components/automate-ui/src/app/helpers/auth/regex.ts

```diff
@@ -23,7 +23,8 @@ export class Regex {
     NO_MIXED_WILDCARD_ALLOW_SPECIAL: '^(\\*|[^:*]+)$',
 
     // Allows valid FQDN only
-    VALID_FQDN: /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-zA-Z0-9_]+([\-\.]{1}[a-zA-Z0-9]+)*\.[a-zA-Z]{2,5}(:[0-9]{1,5})?(\/.*)?$/,
+    // Top level domain is limited to a maximum number of 25 characters
+    VALID_FQDN: /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-zA-Z0-9_]+([\-\.]{1}[a-zA-Z0-9]+)*\.[a-zA-Z]{2,25}(:[0-9]{1,5})?(\/.*)?$/,
```

Comment:
This change from `{2,5}` to `{2,25}` allows a user to utilize a top level domain in their FQDN longer than 5 characters.

For example, previously a user was limited to something like `chef.net` or `chef.times` and would get a error if they used `chef.cooking`.  Now, we can go as high as 25 characters. i.e. `chef.itspizzanight`